### PR TITLE
Add Arengu progressive profiling rule

### PR DESCRIPTION
**Pull request by jacobovidal**
### Description

Add Arengu progressive profiling rule to Auth0's marketplace.

### Testing

- [ ] This change adds test coverage for new/changed/fixed functionality

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [ ] The correct base branch is being used, if not `master`

### CODE REVIEW

File: src/rules/arengu-progressive-profiling.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ * @title Arengu Progressive Profiling 
+ * @overview Capture new users' information in your authentication flows.
+ * @gallery true
+ * @category marketplace
+ */
+
+ async function arenguCompleteUserProfile(user, context, callback) {
+  if (
+    !configuration.SESSION_TOKEN_SECRET ||
+    !configuration.ARENGU_POLICIES_FORM_URL
```
Comment (joshcanhelp) [inline src/rules/arengu-progressive-profiling.js:11]:
```suggestion
    !configuration.ARENGU_PROFILE_FORM_URL
```

File: src/rules/arengu-progressive-profiling.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    try {
+      return ruleUtils.validateSessionToken();
+    } catch (error) {
+      callback(error);
+    }
+  }
+
+  // Modify your login criteria to your needs
+  function isLogin() {
+    return context.stats.loginsCount > 2;
```
Comment (joshcanhelp) [inline src/rules/arengu-progressive-profiling.js:40]: I would suggest the `2` here being a Rule configuration, like:

```
function isLogin() {
  const loginCountForProfiling = configuration.ARENGU_PROFILE_LOGIN_COUNT || 2;
  return context.stats.loginsCount > parseInt(loginCountForProfiling, 10);
}
```

**Review by joshcanhelp — CHANGES_REQUESTED** [review]
Minor fix and one consideration. Otherwise, looks great!

File: src/rules/arengu-progressive-profiling.js (hunk 1 above)
Comment (jacobovidal) [reply src/rules/arengu-progressive-profiling.js]: Fixed!

File: src/rules/arengu-progressive-profiling.js (hunk 2 above)
Comment (jacobovidal) [reply src/rules/arengu-progressive-profiling.js]: Good one! Done!

**Review by joshcanhelp — APPROVED** [review]

**Pull request merged**